src/pages/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,306 @@
+import { Box, Typography, useTheme, Link, Button, IconButton, type Theme } from '@mui/material';
+import React, { useState } from 'react';
+
+import FacebookStatic from '../../assets/facebook.svg';
+import GithubStatic from '../../assets/github.svg';
+import GoogleStatic from '../../assets/google.svg';
+
+import {
+  getPageContainerStyles,
+  getFormStyles,
+  getLogoStyles,
+  getInputStyles,
+  getButtonStyles,
+  forgotPasswordLinkStyles,
+  signUpLinkStyles,
+  orYouCanTextStyles,
+  getSocialIconStyles,
+  legalTextStyles,
+  legalLinkStyles,
+  getErrorTextStyles,
+  GEOMETRY_PX,
+} from './LoginPage.styles';
+
+import type { LoginPageProps } from './LoginPage.types';
+import type { SxProps } from '@mui/material/styles';
+
+type LogoFallbackProps = {
+  orientation?: string;
+  width?: string | number;
+  height?: string | number;
+  alt?: string;
+};
+
+type FormInputFallbackProps = {
+  label?: string;
+  value?: string;
+  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  placeholder?: string;
+  errorMessage?: string;
+  sx?: SxProps<Theme>;
+  type?: string;
+};
+
+type PrimaryButtonFallbackProps = {
+  children?: React.ReactNode;
+  onClick?: (e: React.FormEvent) => void;
+  disabled?: boolean;
+  loading?: boolean;
+  sx?: SxProps<Theme>;
+};
+
+const FacebookIcon: string | undefined = FacebookStatic ?? undefined;
+const GithubIcon: string | undefined = GithubStatic ?? undefined;
+const GoogleIcon: string | undefined = GoogleStatic ?? undefined;
+
+const Logo: React.ComponentType<LogoFallbackProps> = () => <div data-testid="logo-fallback" />;
+
+const FormInput: React.FC<FormInputFallbackProps> = (props) => (
+  <Box
+    component="input"
+    type={props.type || 'text'}
+    placeholder={props.placeholder}
+    value={props.value}
+    onChange={props.onChange}
+    sx={{
+      ...props.sx,
+      '::placeholder': {
+        color: 'text.secondary',
+      },
+    }}
+  />
+);
+
+const PrimaryButton: React.FC<PrimaryButtonFallbackProps> = (props) => (
```

Comment:
PrimaryButton fallback does not expose a 'type' prop and renders a MUI <Button> without type. In a form the button should be type="submit" to support native form submission and keyboard activation. Relying on onClick handler only reduces accessibility and breaks expected form semantics (e.g., Enter key).

**Sugerencia:**
```typescript
Accept and pass a type prop in PrimaryButtonFallbackProps and set <Button type={props.type ?? 'button'} .../>. In the usage inside the form, prefer type="submit" on the primary button and rely on the form's onSubmit handler rather than calling handleSubmit via onClick.
```